scripts/qa_crowbarsetup.sh

```diff
@@ -1064,6 +1074,12 @@ EOF
     local netfile="/etc/crowbar/network.json"
 
     local netfilepatch=`basename $netfile`.patch
+
+    if (( $want_ipv6 > 0 )); then
+        ensure_packages_installed patch
+        patch -p1 $netfile < $scripts_lib_dir/ipv6/network.json.patch
```

Comment:
So what you've just ensured here is no one will be able to patch the network.json since they'll work from the base one and then the patch will fail. I'd *much* prefer this was a different network.json file entirely.